test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -159,7 +159,7 @@ describe('Create token, approve token and approve token without gas', function (
 
         const defaultSpendingCap = await driver.findElement({
           text: '7 TST',
-          css: '.box--flex-direction-row > h6',
+          css: '.mm-box > h6',
```

Comment:
Suggestion & None blocking: can we add a test-id to this element so it won't be depending on dynamic arbitrary style-based classname?

Reply:
Great call out. I admit this was a lazy fix. I've created an issue to fix after this has been merged https://github.com/MetaMask/metamask-extension/issues/20099